### PR TITLE
Wrote editlab view and html

### PR DESCRIPTION
**Pull request by BMHebert**
Implemented a view and html page for viewing and editing labs that a supervisor can access. Also miscellaneous tiny changes.

### CODE REVIEW

**Comment by BMHebert** [comment]
Full list of changes since I was somewhat vague before:

- Heavily editlab.html to display database information and implemented edit functionality.
- Wrote view for editlab with functionality to process input from editlab.html
- Fixed some small formatting issues in edituser.html
- Added __str__ method for UserProfile
- Fixed LabManagement.editLab since it wasn't properly saving the object to the database.
- Improved CourseManagement.inputErrorChecker since it wasn't really written with much care in the first case.

**Review by WajahatAliShah1 — APPROVED** [review]
All changes look good, once second review is in, we can merge and update our tests to test functionality of saving new or edited information to database.

**Pull request merged**
